docs/gen2_Tutorial.md

```diff
@@ -434,19 +434,19 @@ To create a gen2 cluster with autoscaler, we need:
 
 - bundle pointing to the bundle/capi-cluster-autoscaler in the arlon repository.
 - dynamic profile that contains the above bundle.
-- a basecluster manifest(that makes use of MachineDeployment and not MachinePools) which has the CAPI annotations for min and max nodes set ( as a part of `preparegit` or manually add it ).
-- run `arlon cluster create` with the repo-path pointing to the basecluster manifest described in the step above, set the profile to  be the one created in step 2 and pass the `autoscaler` flag.
+- a cluster template manifest(that makes use of MachineDeployment and not MachinePools) which has the CAPI annotations for min and max nodes set ( as a part of `preparegit` or manually add it ).
+- run `arlon cluster create` with the repo-path pointing to the cluster template manifest described in the step above, set the profile to  be the one created in step 2 and pass the `autoscaler` flag.
 
 ### Bundle creation
 
 Register a dynamic bundle pointing to the bundles/capi-cluster-autoscaler in the Arlon repo.
 
-To enable the cluster-autoscaler bundle, add one more parameter during cluster creation: `srcType`. This is the ArgoCD-defined application source type (Helm, Kustomize, Directory).
+To enable the cluster-autoscaler bundle, add one more parameter during cluster creation: `srcType`. This is the ArgoCD-defined application source type (Helm, Kustomize, Directory). In addition to this, the `repo-revision` parameter should also be set to a stable arlon release branch ( in this case v0.9 ).
 
 This example creates a bundle pointing to the bundles/capi-cluster-autoscaler in Arlon repo
 
 ```shell
-arlon bundle create cas-bundle --tags cas,devel,test --desc "CAS Bundle" --repo-url https://github.com/arlonproj/arlon.git --repo-path bundles/capi-cluster-autoscaler --srctype helm --repo-revision v0.9
+arlon bundle create cas-bundle --tags cas,devel,test --desc "CAS Bundle" --repo-url https://github.com/arlonproj/arlon.git --repo-path bundles/capi-cluster-autoscaler --srctype helm --repo-revision v0.10
```

Comment:
v0.10 here but a paragraph earlier it says v0.9?

Reply:
Done